phases/ephemeral/docs.md

```diff
@@ -1190,7 +1190,12 @@ A buffer into which to write the preopened directory name.
 
 #### <a href="#pwrite" name="pwrite"></a> `pwrite(fd: fd, iovs: ciovec_array, offset: filesize) -> (errno, size)`
 Write to a file descriptor, without using and updating the file descriptor's offset.
-Note: This is similar to `pwritev` in POSIX.
+Note: This is similar to `pwritev` in Linux (and other Unix-es).
+
+Like Linux (and other Unix-es), [`pwrite`](#pwrite) should write all data in the `$fd` is
+for a regular file or a symbolic link to a regular file: Any calls of [`pwrite`](#pwrite)
```

Comment:
Is `in` meant to be `if` here?

Also, POSIX doesn't say that `pwrite` etc. will write *all* the data; they could still be interrupted in the middle if an error occurs, such running out of space on the output device.

Also, as a minor correction, POSIX resolves symbolic links at `open` time, so it's not necessary to call out symblic links here in `pread` and `pwrite`. The reason POSIX mentions symbolic links in the paragraph linked above is that the list of affected functions includes `readlink` and `symlink` which operate on symbolic links themselves.